dev/tpv2/run.sh

```diff
@@ -16,5 +16,5 @@
 set -o errexit
 
 cd "$TC/experimental/traffic-portal"
-npm ci
-./node_modules/.bin/ng serve --ssl --ssl-cert /server.crt --ssl-key /server.key --watch --progress --proxy-config "$TC/dev/tpv2/proxy.json" --port 443 --host "::0" --live-reload
```

Comment:
`--progress` is "unknown"? I see it in the output of `ng serve --help`:

```
  --progress 
    Log progress to the console while building.
```

Reply:
What version of angular cli are you using? Perhaps your node-modules is out of date (might need to nuke it). I'm using 13.3.1 and `ng seve --help | grep prog` returns nothing

In the TPv2 container:
```
trafficcontrol-tpv2-1  | npm notice
trafficcontrol-tpv2-1  | npm notice New minor version of npm available! 8.5.5 -> 8.6.0
trafficcontrol-tpv2-1  | npm notice Changelog: <https://github.com/npm/cli/releases/tag/v8.6.0>
trafficcontrol-tpv2-1  | npm notice Run `npm install -g npm@8.6.0` to update!
trafficcontrol-tpv2-1  | npm notice
trafficcontrol-tpv2-1  | Node.js version v17.8.0 detected.
trafficcontrol-tpv2-1  | Odd numbered Node.js versions will not enter LTS status and should not be used for production. For more information
, please see https://nodejs.org/en/about/releases/.
trafficcontrol-tpv2-1  | Unknown option: '--progress'
```

Reply:
Looks like it's deprecated in [Angular 12](https://v12.angular.io/cli/serve) and removed in [Angular 13](https://angular.io/cli/serve). Curiously, `update.angular.io` doesn't list this as a breaking change.

Reply:
I was in a different project directory, which uses Angular 11.